england.qmd

```diff
@@ -0,0 +1,492 @@
+---
+title: "Structural inequalities in General Practice in England"
+brand: _brand.yml
+---
+
+```{r setup, include=FALSE}
+# Load necessary libraries
+library(ggplot2)
+library(showtext)
+library(tidyverse)
+library(magrittr)
+
+# Set up Google font and automatic display for text rendering
+showtext_auto()
+
+# Set up the ggplot theme
+theme_set(
+  theme_minimal() +
+    theme(
+      axis.title = element_text(size = 80),
+      axis.text = element_text(size = 80),
+      plot.caption = element_text(size = 60),
+      plot.title = element_text(size = 100),
+      plot.subtitle = element_text(size = 90),
+      panel.grid = element_line(size = 5),
+      legend.title = element_text(size = 60),
+      legend.text = element_text(size = 60),
+      legend.key.width = unit(3.5, "cm"),
+      plot.title.position = "plot"
+    )
+)
+
+update_geom_defaults("point", list(size = 50))
+
+df <- read.csv("https://raw.githubusercontent.com/HealthEquityEvidenceCentre/HEEC/refs/heads/main/datapacks/final_data.csv")
+df[is.na(df$quin_4) & is.na(df$quin_5), ]
+df %<>%
+  mutate(quin_5 = ifelse(is.na(quin_5), quin_4, quin_5))
+
```

Comment:
If quin_5 is NA (missing), it assigns the value from quin_4 to quin_5. If quin_5 is not NA, it retains its value.

This shouldn't be necessary for this national level analysis, so I can remove it (or comment it out).

I'm not sure what line 50 refers to (I think the line numbers have changed), but the line was only run once as it applied to all indicators across all ICBs. This is so that if there were no practices in the most deprived quintile in the ICB, there would still be a datapoint representing the most deprived practices **in that ICB** (not the country)

Reply:
Ah sorry, you are right about the lines being out of synch - that was local to me! I think it's good to keep the ICB level code in but comment it out and leave a note about that because it's highly likely if anyone wishes to replicate this for their area they will need this logic.